### DISCUSSION TITLE
Kiosk Mode for Raspberry Pi 7 Inch Screen

### DISCUSSION THREAD

**Discussion by tim273** [poblabs/weewx-belchertown#839] [Ideas]
While there is a kiosk mode for a 3.5 inch screen, I would love to see another one that is optimized for the Raspberry Pi 7 inch screen.  I have set it up before, but I had to set a zoom value on it and it was OK, but could be better.

**Comment by ThePeePs** [comment]
There is the kiosk.html page that had added a while back, but it might be bit big for a 7 inch screen. as it's made to fit in a 1200x800 px space.

**Reply by tim273** [reply]
Yes, and I did use that, I was thinking it would be nice to have one that's more geared toward that size screen, more optimized for it that is.
